File: despatch.WEB/app/src/swagger/models/contact-data-addresses-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { AddressResponse } from './address-response';
import { ContactDataResponse } from './contact-data-response';
export interface ContactDataAddressesResponse extends BaseEntityResponse{
  address?: AddressResponse;
  addressId?: string;
  contactData?: ContactDataResponse;
  contactDataId?: string;
}

File: despatch.Domain/Models/Default/Response/ContactDataAddressesResponse.cs
using despatch.Domain.Enums;

namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataAddressesResponse : BaseEntityResponse
    {
        public string AddressId { get; set; }
        public string ContactDataId { get; set; }

        public virtual AddressResponse Address { get; set; }
        public virtual ContactDataResponse ContactData { get; set; }
    }
}
